htdocs/compta/facture/card.php

```diff
@@ -302,6 +302,15 @@
 		// Validation
 		$object->fetch($id);
 
+		if (!empty($conf->global-> INVOICE_CHECK_POSTERIOR_DATE)) {
+			$last_of_type = $object->willBeLastOfSameType();
+			if (empty($object->date_validation) && !$last_of_type[0]) {
+				setEventMessages($langs->transnoentities("ErrorInvoiceIsNotLastOfSameType", $object->ref, dol_print_date($object->date, 'day'), dol_print_date($last_of_type[1], 'day')), null, 'errors');
+				header('Location: '.$_SERVER["PHP_SELF"].'?facid='.$id);
```

Comment:
Why making a redirect here ? and not just 
$action = '';
like done on other tests after ?

Reply:
I wanted to stop rendering the page to prevent the validation. But you are right, using `$action = ''` has the same effect. I just pushed a new commit with this modification.